manifests/namespace-install-with-hydrator/kustomization.yaml

```diff
@@ -0,0 +1,3 @@
+resources:
+  - ../cluster-install
```

Comment:
Is this correct? shouldn't it be namepsace-install

Reply:
+1